packages/react-client/src/hooks/useFishjamClientState.ts

```diff
@@ -44,41 +43,12 @@ const eventNames = [
 ] as const satisfies (keyof MessageEvents<unknown, unknown>)[];
 
 export interface FishjamClientState {
-  peers: PeerState[];
-  components: PeerState[];
-  localPeer: PeerState | null;
+  peers: Record<string, Peer<PeerMetadata, TrackMetadata>>;
```

Comment:
Nit: There is an alias for the `string` type in `types/interna.ts` that indicates this particular string is a `PeerId`